craft_parts/sources/git_source.py

```diff
@@ -287,7 +287,7 @@ def _format_source(self) -> str:
         Local sources are formatted as file:///absolute/path/to/local/source
         This is done because `git clone --depth=1 path/to/local/source` is invalid
         """
-        protocol_pattern = re.compile(r"^[\w\-.@]+:")
+        protocol_pattern = re.compile(r"^[\w\-.@+]+:")
```

Comment:
Can you update the comment? 
`Protocols for git are http[s]://, ftp[s]://, git://, ssh://, and file://.` -> `Protocols for git are http[s]://, ftp[s]://, git://, ssh://, git+ssh://, and file://.`

Reply:
Updated to include git+ssh://.